### PR TITLE
fixed cannot run test-app script

### PR DESCRIPTION
**Pull request by nndung179**
**Problem:**
+ I pulled webpack-block to my local machine. 
+ cd to test-app
+ After run `npm i`, I could not run `npm start`, it gave me message below

```
(function (exports, require, module, __filename, __dirname) { const {
                                                                    ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:404:25)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
```

**Solution**
+ I have figured it out that the config of webpack in `test-app` uses syntax of JS ES6. So that, I decided to change file name of webpack config from: **webpack.config.js** to **webpack.config.babel.js** and change scripts in package.json. Now it worked well

### CODE REVIEW

**Comment by andywer** [comment]
Hey @nndung179 for your PR! Good catch.

But don't we need `babel-register`, too now?

**Comment by nndung179** [comment]
I discovered that. When you run `npm i`. Look in insider **node_modules**, it was already there.

**Comment by andywer** [comment]
Ok, I see that `babel-core` requires `babel-register`, so it will work with node 5+.

The problem is: I tried it locally (changing the file name to `webpack.config.babel.js`) and it still fails with the same error... Did you do something else?

The other way to fix it would be to just get rid of the object destructuring assignment in the first three lines.

**Comment by andywer** [comment]
Or... another quick-fix, since it's just the test app:

We could just add `{ "engines": { "node": ">= 6.0" } }` to the `package.json`... 🤔

**Comment by nndung179** [comment]
I think I will stick with the first one: replace **destructuring** to **object.attribute**

**Comment by andywer** [comment]
@nndung179 Nevermind. My mistake that it didn't work for me locally... 🙈😉

Let's take your fix and just add `{ "engines": { "node": ">= 5.0" } }` to the `package.json`, since it will not work with node 4.x (because we need "flat" node_modules, which requires npm 3.x which ships with node 5+) :)

File: test-app/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
-    "build": "NODE_ENV=production webpack --config webpack.config.js",
-    "start": "NODE_ENV=development webpack-dev-server --config webpack.config.js"
+    "build": "NODE_ENV=production webpack --config webpack.config.babel.js",
+    "start": "NODE_ENV=development webpack-dev-server --config webpack.config.babel.js"
   },
   "devDependencies": {
```
Comment (andywer) [inline test-app/package.json]: Please add `{ "engines": { "node": ">= 5.0" } }` and this is ready for merging :)

Comment (nndung179) [inline test-app/package.json]: ok right a way

**Comment by andywer** [comment]
👍👍

I just re-ordered stuff in the package.json as I recognized I can actually commit changes from within this PR site :)

**Comment by nndung179** [comment]
I just known too :). Thank mate!

**Pull request merged**